components/SoftwareDetails.tsx

```diff
@@ -32,15 +55,27 @@ type SoftwareDetailsProps = {
 const SoftwareDetails = ({ detailItems, className }: SoftwareDetailsProps) => (
   <div
     className={cn(
-      "group/software grid grid-cols-[1fr,4fr,auto,4fr,1fr] gap-8 p-8",
+      "group/software grid gap-4 lg:gap-8 p-4 lg:p-8",
+      "lg:grid-cols-[2fr,4fr,1fr,auto,1fr,4fr,2fr]",
+      "grid-cols-[0,4fr,1fr,auto,1fr,4fr,0]",
```

Comment:
The first and last column are effectively padding from the 2nd row down... on mobile I trim these to 0 and reduce the gap to allow the other metrics to have a bit more space (positions 1 through 6 below)